src/components/Contentful/JourneyCardCarousel.vue

```diff
@@ -177,9 +189,22 @@ const isNonBadgeSlide = slide => {
 
 const orderedSlides = computed(() => {
 	const achievementSlides = [];
+	let loanJourneys = [];
+	let sortedSlides = [];
+
+	const transactionLoans = props.userInfo?.transactions?.values?.filter(t => {
```

Comment:
Is `transactions` returning all transactions for the user? What happens if the users has 1000s of transactions?